siriuspy/siriuspy/clientconfigdb/types/as_rf.py

```diff
@@ -452,50 +452,87 @@ def get_dict():
-    ['RA-ToSIA02:AlarmConfig:InnerPowerLimLow', 0, 0.0],
-    ['RA-ToSIA02:AlarmConfig:InnerPowerLimLoLo', 0, 0.0],
-    ['RA-ToSIA02:AlarmConfig:CurrentLimHiHi', 0, 0.0],  # Alarms lims currents
-                                                        # SSA tower 2
-    ['RA-ToSIA02:AlarmConfig:CurrentLimHigh', 0, 0.0],
-    ['RA-ToSIA02:AlarmConfig:CurrentLimLow', 0, 0.0],
-    ['RA-ToSIA02:AlarmConfig:CurrentLimLoLo', 0, 0.0],
+    # NOTE: Alarms and offset of SSA towers 1 & 2 temporaly removed
+    # ['RA-ToSIA01:OffsetConfig:UpperIncidentPower', 0, 0.0],  # Offsets SSA Twr1
+    # ['RA-ToSIA01:OffsetConfig:UpperReflectedPower', 0, 0.0],
+    # ['RA-ToSIA01:OffsetConfig:LowerIncidentPower', 0, 0.0],
+    # ['RA-ToSIA01:OffsetConfig:LowerReflectedPower', 0, 0.0],
+    # ['RA-ToSIA01:OffsetConfig:InputIncidentPower', 0, 0.0],
+    # ['RA-ToSIA01:OffsetConfig:InputReflectedPower', 0, 0.0],
+    # ['RA-ToSIA01:OffsetConfig:OutputIncidentPower', 0, 0.0],
+    # ['RA-ToSIA01:OffsetConfig:OutputReflectedPower', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:GeneralPowerLimHiHi', 0, 0.0],  # Alarms lims pwr SSA tower 1
+    # ['RA-ToSIA01:AlarmConfig:GeneralPowerLimHigh', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:GeneralPowerLimLow', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:GeneralPowerLimLoLo', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:InnerPowerLimHiHi', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:InnerPowerLimHigh', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:InnerPowerLimLow', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:InnerPowerLimLoLo', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:CurrentLimHiHi', 0, 0.0],  # Alarms lims currents SSA tower 1
+    # ['RA-ToSIA01:AlarmConfig:CurrentLimHigh', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:CurrentLimLow', 0, 0.0],
+    # ['RA-ToSIA01:AlarmConfig:CurrentLimLoLo', 0, 0.0],
+    # ['RA-ToSIA02:OffsetConfig:UpperIncidentPower', 0, 0.0],  # Offsets SSA Twr2
+    # ['RA-ToSIA02:OffsetConfig:UpperReflectedPower', 0, 0.0],
+    # ['RA-ToSIA02:OffsetConfig:LowerIncidentPower', 0, 0.0],
+    # ['RA-ToSIA02:OffsetConfig:LowerReflectedPower', 0, 0.0],
+    # ['RA-ToSIA02:OffsetConfig:InputIncidentPower', 0, 0.0],
+    # ['RA-ToSIA02:OffsetConfig:InputReflectedPower', 0, 0.0],
+    # ['RA-ToSIA02:OffsetConfig:OutputIncidentPower', 0, 0.0],
+    # ['RA-ToSIA02:OffsetConfig:OutputReflectedPower', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:GeneralPowerLimHiHi', 0, 0.0],  # Alarms lims pwr SSA tower 2
+    # ['RA-ToSIA02:AlarmConfig:GeneralPowerLimHigh', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:GeneralPowerLimLow', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:GeneralPowerLimLoLo', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:InnerPowerLimHiHi', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:InnerPowerLimHigh', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:InnerPowerLimLow', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:InnerPowerLimLoLo', 0, 0.0],
+    # ['RA-ToSIA02:AlarmConfig:CurrentLimHiHi', 0, 0.0],  # Alarms lims currents SSA tower 2
```

Comment:
you have to break lines here, otherwise it will be larger than 80 columns.

Reply:
Ok! I wasn't aware of this limit. I will rewrite it as it was.

Reply:
see response to comment below.